src/main.js

```diff
@@ -91,10 +91,32 @@ function createModulesTableCell(text) {
 /**
  * Create one row of the modules table.
  */
-function createModulesTableRow(parsedModule) {
+async function createModulesTableRow(parsedModule) {
     let tr = document.createElement('tr');
     tr.appendChild(createModulesTableCell(parsedModule.name));
-    tr.appendChild(createModulesTableCell(parsedModule.moduleType));
+
+    // get correct localization for the moduletype
+    switch (parsedModule.moduleType) {
+        case "Kernmodul":
+            tr.appendChild(createModulesTableCell(await i18n.getMessage("coreModule")))
+            break;
+        case "Projektmodul":
+            tr.appendChild(createModulesTableCell(await i18n.getMessage("projectModule")))
+            break;
+        case "Zusatzmodul":
+            tr.appendChild(createModulesTableCell(await i18n.getMessage("additionalModule")))
+            break;
+        case "Erweiterungsmodul":
+            tr.appendChild(createModulesTableCell(await i18n.getMessage("extensionModule")))
+            break;
+        case "Majormodul":
+            tr.appendChild(createModulesTableCell(await i18n.getMessage("majorModule")))
+            break;
+        default:
+            tr.appendChild(createModulesTableCell(parsedModule.moduleType))
+            break;
+    }
```

Comment:
just call the key in the JSON file like the `parsedModule.moduleType` and you don't need this code.
```suggestion
    tr.appendChild(createModulesTableCell(await i18n.getMessage(parsedModule.moduleType)))
```

Reply:
We need this, because we use the German API. So the value of `parsedModule.moduleType` are the German words. Like Kernmodul and not the needed English variants core module  used in the 18n JSON files.
Or am I mistaken?

Reply:
I'd prefer having german keys in the JSON file instead of this monstrosity of a switch statement.